meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,296 @@
+  "genre": ["adventure"],
+  "synopsis": "A man must overcome trauma and fight off a mysterious assassin."
+}
+```
+And the following query:
+```json
+{
+  "q": "adventure",
+  "attributesToSearchOn": ["genre"]
+}
+```
+Meilisearch will only return document 1.
+
+Both documents contain the word adventure, but `"attributesToSearchOn": ["genre"]` instructs Meilisearch to only consider results found on the genre field.
+
+
+Done by @ManyTheFish in (#3834)
+
+
+## Search for facet values
+Meilisearch v1.3.0 introduces a new endpoint  `POST /indexes/{index}/facet-search` that allows you to search for facet values. Only fields defined as `filterableAttributes` will be facet-searchable. Facet search supports [prefix search](https://www.meilisearch.com/docs/learn/advanced/prefix) and [typo tolerance](https://www.meilisearch.com/docs/learn/configuration/typo_tolerance).
+
+```bash
+ curl \
+  -X POST 'http://localhost:7700/indexes/movies/facet-search' \
+  -H 'Content-Type: application/json'  \
+--data-binary '{
+    "facetName": "genres",
+    "facetQuery": "a"
+  }'
+
+```
+
+Done by @Kerollmops in (#3699)
+
+## Sort facet values by count
+By default, all facet values are sorted in ascending alphanumeric order. 
+```json
+"sortFacetValuesBy": {"*": "alpha"}
+```
+
+With the release of v1.3.0 you can now order by `count` using the `sortFacetValuesBy` property of the [`faceting` index settings](https://www.meilisearch.com/docs/reference/api/settings#faceting). This allows you to sort facet values in descending order by the number of documents containing that facet value.
+
+It is possible to change this ordering for all facets using `*`:
+
+```bash
+ curl \
+  -X PATCH 'http://localhost:7700/indexes/movies/settings/faceting \
+  -H 'Content-Type: application/json'  \
+--data-binary '{
+    "sortFacetValuesBy": {"*": "count"}
+  }'
+
+```
+
+Alternatively, it is possible to specify ordering per facet i.e. order the genre facet by count, for everything else apply alphanumeric ordering.
+
+```bash
+ curl \
+  -X PATCH 'http://localhost:7700/indexes/movies/settings/faceting \
+  -H 'Content-Type: application/json'  \
+--data-binary '{
+    "sortFacetValuesBy": {"*": "alpha", "genre": 'count'}
+  }'
+
+```
+
+Done by @Kerollmops in (#3612)
+
+### Language support improvements
+
+<!-- Charabia changelogs from 0.7.2 onwards: -->
+* Transform classifier into normalizer (#214) @ManyTheFish 
+* Update Lindera to 0.24.0 (#212) @mosuka
+* Handle underscore similarly to dash (#216) @vvv 
+* Add UniDic implimentation (#218) @mosuka
+* Bump peter-evans/create-pull-request from 4 to 5 (#213)
+* Bump Swatinem/rust-cache from 2.2.1 to 2.4.0 (#217)
+
+
+### Other improvements
+* Improve the metrics. Provides metrics on the task queue such as the number of queued tasks and the number of processing tasks. Adds the real database size used by Meilisearch. (#3789) @irevoire
+* Add "meilisearch" prefix to last metrics that were missing it (#3861) @dureuill 
```

Comment:
Can't this line be part of the prometheus feature improvement?

Reply:
oh these were hidden and I missed them! yea it was a different PR but definitely could be. I'll adjust

Reply:
thats changed